docs/en/observability/synthetics-configuration.asciidoc

```diff
@@ -212,11 +212,7 @@ Control whether or not to capture screenshots. Options include `'on'`, `'off'`,
 `alert` (https://github.com/elastic/synthetics/blob/{synthetics_version}/src/dsl/monitor.ts#L47-L54[`AlertConfig`])::
 Enable or disable alerts. Read more about alerts in <<synthetics-settings-alerting>>.
 +
-`status.enabled` (`boolean`):::
-Enable monitor status alerts.
-+
-`tls.enabled` (`boolean`):::
-Enable TLS certificate alerts.
+`alert: {status: {enabled: [true|false]}}`
```

Comment:
I wonder if we reformat lines 212-215 so:

1. We can remove [the link to `AlertConfig`](https://github.com/elastic/synthetics/blob/v1.5.1/src/dsl/monitor.ts#L47-L54) since it sounds like that's not the correct type (since the `tls` portion is not supported).
2. We can make it clear that this is for _monitor status_ alerts only.

```
`alert.status.enabled` (`boolean`)::
Enable or disable monitor status alerts. Read more about alerts in <<synthetics-settings-alerting>>.
```

Reply:
Good idea, thanks Collen - have used that